backend/package.json

```diff
@@ -16,7 +16,8 @@
     "mempool",
     "blockchain",
     "explorer",
-    "liquid"
+    "liquid",
+    "lightning"
```

Comment:
We don't need this `lightning` dep anymore right?

Reply:
This one is an npm keyword/search word, the deps are a few more lines down

Reply:
Oh my bad!